src/main/java/seedu/address/model/person/predicates/AddressContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,32 @@
+
+package seedu.address.model.person.predicates;
```

Comment:
Is there a way to make all these ContainsKeyword class general (into one class or something like that) so that you don't need to write a class for all the criteria under checking? Or do you think splitting into multiple classes is better?